cob_hardware_config/common/cob4_driver_config.yaml

```diff
@@ -0,0 +1,52 @@
+sync:
+  interval_ms: 20
+  overflow: 0
+heartbeat:
+  rate: 20
+  msg: "77f#05"
+
+defaults:
+  eds_pkg: cob_hardware_config
+  eds_file: "common/Elmo.dcf"
+  dcf_overlay:
+    "6083": "1000000"
+    "6084": "1000000"
+    "60C5": "1000000"
+    "60C6": "1000000"
```

Comment:
can we add comments about meaning and units from the elmo docu for these objects?